app/javascript/components/upload/WizardNavPanel.jsx

```diff
@@ -184,7 +184,7 @@ function MainStepsDisplay(formState, serverState, currentStep, setCurrentStep, m
           </span>
           <span>
             <a className="action link" role="link">
-            AnnData <FontAwesomeIcon icon={expansionIcon}/>
+            AnnData <sup>BETA</sup><FontAwesomeIcon icon={expansionIcon}/>
```

Comment:
Fair question, it seems like in general we don't have a standard on using tags like BETA and it might be a good idea to revisit this. There's some good conversation in this thread on slack: https://broadinstitute.slack.com/archives/C024HV2FANL/p1677613076594829?thread_ts=1677601938.185519&cid=C024HV2FANL